test/server/cards/03_TWI/units/KrakenConfederateTactician.spec.ts

```diff
@@ -0,0 +1,68 @@
+describe('Kraken Confederate Tactician', function () {
+    integration(function (contextRef) {
+        describe('Kraken Confederate Tactician\'s ability', function () {
+            it('should create two friendly Battle Droid tokens', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['kraken#confederate-tactician'],
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.kraken);
+
+                const battleDroids = context.player1.findCardsByName('battle-droid');
+                expect(battleDroids.length).toBe(2);
+                expect(battleDroids).toAllBeInZone('groundArena');
+                expect(battleDroids.every((battleDroid) => battleDroid.exhausted)).toBeTrue();
+                expect(context.player2.getArenaCards().length).toBe(0);
+            });
+
+            it('should give +1/+1 to friendly tokens unit on attack', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['droid-deployment', 'drop-in'],
```

Comment:
I just merged the fix that allows token units to be added in `setupTest` so you can do that now instead of needing to sue these